include/build_id.hpp

```diff
@@ -0,0 +1,17 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0. This product includes software
+// developed at Datadog (https://www.datadoghq.com/). Copyright 2021-Present
+// Datadog, Inc.
+
+#include "span.hpp"
+
+#include <string>
+#include <vector>
+
+namespace ddprof {
+using BuildIdSpan = ddprof::span<const unsigned char>;
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏you might want to use `const std::byte` instead of `const unsigned char` to make explicit that these are not characters

Reply:
Thanks, c++ 17 things !

Reply:
What went wrong ?